{"uuid": "9406f518-dcf5-4fc2-a6e2-3710a1447dad", "children": ["00091473-301c-44d4-9515-db5b0209a907"], "befores": [{"name": "setup_class", "status": "passed", "start": 1751550749112, "stop": 1751550754337}], "afters": [{"name": "setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'request' is not defined\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/allure_commons/_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/_pytest/fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"/Users/pinky/Documents/PythonTest/tests/base_test.py\", line 24, in setup_class\n    if hasattr(request.node, \"rep_call\") and request.node.rep_call.failed:\n               ^^^^^^^\n"}, "start": 1751550776912, "stop": 1751550776913}], "start": 1751550749112, "stop": 1751550776913}